apps/generator/docs/nunjucka-depreciate.md

```diff
@@ -0,0 +1,120 @@
+--
+title: "Depreciation of nunjucks render engine"
```

Comment:
```suggestion
title: "Depreciation of Nunjucks render engine"
```

Reply:
also spelling error in file name

Reply:
title is a migration guide not deprecation notice

@Florence-Njeri shouldn't it sound completely different?

Reply:
Hey @derberg , you're right the title is wrong